README.md

```diff
@@ -2,23 +2,163 @@
+- `/z.dol` - Swiss
+- `/b.dol` - GBI
+
+**Pro-tip:** You can prevent files from showing in Swiss by marking them as hidden files on the SD card.
+
+If you hold multiple buttons, the highest in the table takes priority. Also note that all directions are D-Pad buttons, *not* joysticks. Careful not to touch any of the analog controls (sticks and triggers) when powering on as this is when they are calibrated.
+
+**Something not working?** See the [troubleshooting section](#troubleshooting).
+
+
 ## Usage
 
-Flash the latest build to your Qoob Pro or Viper as a BIOS.  
-The Qoob SX is currently not supported because it uses a very different process
-to boot, and reverse-engineering efforts so far have been unsuccessful.
+Download the designated appropriate file for your device from the [latest release](https://github.com/redolution/iplboot/releases/latest).
+
+Prepare your SD card by copying DOLs onto the SD card and renaming them according the table above.
+
+### PicoBoot
+
+You can download ready-to-go firmware featuring IPLBoot directly from PicoBoot. Just follow the [installation guide](https://github.com/webhdx/PicoBoot#-installation-guide). This is likely what you want.
+
+Otherwise, use `iplboot.dol` and follow the PicoBoot [compiling firmware](https://github.com/webhdx/PicoBoot#compiling-firmware) instructions to flash your Raspberry Pico.
+
+### Qoob Pro
+
+Use `iplboot.gcb` to flash your Qoob Pro as a BIOS.
+
+### Qoob SX
+
+Qoob SX is not currently supported.
+
+### Viper
+
+Use `iplboot.vgc` to flash your Viper as a BIOS.
+
+### GameCube Memory Card
+
+Use `iplboot_xz.gci` and a GameCube memory card manager such as [GCMM](https://github.com/suloku/gcmm) to copy to your memory card. It will be saved as `boot.dol` and can be used in conjunction with the various [save game exploits](https://www.gc-forever.com/wiki/index.php?title=Booting_homebrew#Game_Save_Exploits).
+
+
+## Troubleshooting
+
+IPLBoot displays useful diagnostic messages as it attempts to load the selected DOL. But it's so fast you may not have time to read or even see them. If you hold the down direction on the D-Pad, the messages will remain on screen until you let go.
+
+
+## Compiling
+
+You should only need to compile if you want to modify to the IPLBoot source code. Otherwise, you can just download needed files from the [latest release](https://github.com/redolution/iplboot/releases/latest).
+
+1. Ensure you have the latest version of [devkitPro](https://devkitpro.org/wiki/Getting_Started) installed.
+2. Ensure you have the latest version of [libogc2](https://github.com/extremscorner/libogc2) installed (see [instructions](#installing-libogc2) below).
+3. Ensure your devkitPro environment variables are set: `DEVKITPRO` and `DEVKITPPC`
+
+IPLBoot also acts as a server for @emukidid's [usb-load](https://github.com/emukidid/gc-usb-load), should you want to use it for development purposes.
```

Comment:
Since this is for homebrew development in general and not just for iplboot itself, I think this belongs under usage.